__tests__/app.test.js

```diff
@@ -417,3 +417,28 @@ describe("/api/comments/:comment_id", ()   =>   {
     });
 });
 
+describe("api/users", ()   =>   {
+    describe('GET',  ()   =>  {
+        it('200 status code and array of user objects',  ()  =>  {
+            return request(app)
+            .get("/api/users")
+            .expect(200)
+            .then((response) =>  {
+                const { users } = response.body
+                expect(Array.isArray(users)).toBe(true)
```

Comment:
No need to check for it being an array as we are later using a forEach this proves that it is an array